src/pages/product/ProductDetailInfo/Hashtags.jsx

```diff
@@ -0,0 +1,22 @@
+import React, { Component } from 'react';
+import './Hashtags.scss';
+
+class Hashtags extends Component {
+  render() {
+    const { first_mood, series, seriesNumber, name } = this.props;
+    return (
+      <div className="hashtags">
+        {first_mood &&
+          first_mood.map(mood => (
```

Comment:
이렇게도 사용할 수 있습니다! optional chaining이라고 합니다!

```suggestion
        {first_mood?.map(mood => (
```

Reply:
감사합니다. 수정했습니다!

https://github.com/wecode-bootcamp-korea/fullstack1-1st-BaseNote-frontend/pull/3/commits/aa6a7062899efcf9716a875693e32f4f24c0e482